app-i18n/libhangul/libhangul-0.1.0.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="Library to support hangul input method logic"
+DESCRIPTION="A library to support hangul input method logic, hanja dictionary and small hangul character classification."
+HOMEPAGE="https://github.com/libhangul/libhangul"
+COPYRIGHT="2004-2021 Hwangjin Choe"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/libhangul/libhangul/archive/refs/tags/libhangul-0.1.0.zip"
+CHECKSUM_SHA256="e90fb326167f8d54ba637bf5d1d55cbe9956f6b03a7ae9cb0786037b4a2a487b"
+SOURCE_DIR="libhangul-$portVersionedName"
+
+ARCHITECTURES="all"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libhangul$secondaryArchSuffix = $portVersion
+	cmd:hangul
+	lib:libhangul$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libhangul${secondaryArchSuffix}_devel = $portVersion
+	devel:libhangul$secondaryArchSuffix = $portVersion
+	"
+
+BUILD_REQUIRES="
+	gettext
```

Comment:
gettext not needed here, use devel:libintl, also move haiku_devel here instead of BUILD_PREREQUIRES (same with expat)